src/test/java/com/google/api/codegen/protoannotations/testdata/go_library_with_grpc_service_config.baseline

```diff
@@ -0,0 +1,4705 @@
+============== file: cloud.google.com/go/library/apiv1/UpdateBook_smoke_test.go ==============
```

Comment:
> Do we really need all these new baseline files?

Probably not, especially because this feature is mainly to support languages that don't have a microgenerator and need to continue to use this generator OR to support a migration step for particularly complicated APIs.

> My assumption was that the feature allowed to configure retries from a different source, but the actual result...should remain intact.

Correct.

> The problem with having the whole new set of baseline tests is that they go out of control

I totally agree and they have been a hassle to keep up-to-date with master.

I will keep the Java and PHP baselines because Java will probably stay on this generator for a while and PHP won't have a microgenerator (AFAIK), while all other languages have one.

WDYT?

Reply:
> this PR adds 82,868 new lines of code

I'm trying to bump my contributor ranking :wink: just kidding of course

Reply:
Done.